### PR TITLE
feat: Add recommended VSCode settings

### PR DESCRIPTION
**Pull request by nderjung**
Signed-off-by: Alexander Jung <alex@unikraft.io>

### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,6 @@
 # Ignore distributables folder
 dist/
 
-# Ignore vscode folder
-.vscode/
-!.vscode/settings.json
-
```
Comment (craciunoiuc) [inline .gitignore]: Why remove this, if someone changes their env it will pop up as a changed file.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "files.associations": {
+    "Kraftfile": "yaml",
+  },
+  "editor.rulers": [80],
```
Comment (craciunoiuc) [inline .vscode/settings.json:5]:
```suggestion
  "editor.rulers": [80, 100, 120],
```

I usually use these, we could at least add `100` also

Comment (nderjung) [reply .vscode/settings.json:5]: Most of the files are 80, which I think is preferable, I fear that adding extra may incentive setting at longer markers.

Comment (craciunoiuc) [reply .vscode/settings.json:5]: yeah, that's why there are two: 80 - soft limit, 100 - hard limit :D

**Review by craciunoiuc — APPROVED** [review]
All good here. Thanks!

Reviewed-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>
Approved-by: Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>

**Pull request merged**